### PR TITLE
Add python version 3 in README.md

### LINKED ISSUE #1463: description of script command at README.md needs to be fixed
README.md currently describes about the script command as follows.

> The script command allows user to run a custom script on a data recorded. The supported script types are Python 2.7 and Lua 5.1 as of now.

However, we do support Python 3 along with Python 2.7 so this needs to be updated as follows.

> The script command allows user to run a custom script on a data recorded. The supported script types are Python 3, Python 2.7 and Lua 5.1 as of now.

The same change has to be applied to `doc/ko/README.md`.

### PR DESCRIPTION
**Pull request by daeroro**
fixes #1463

### CODE REVIEW

**Comment by kangtegong** [comment]
Hello, for your information, you have to sign off your commit. 
(See CONTRIBUTING.md for more information)
Please amend the commit :)
Also, I think it's better to have more explanation on your commit mesage.

File: README.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ The `info` command shows system and program information when recorded.
     # disk iops           : 0 / 24 (read / write)
 
 The `script` command allows user to run a custom script on a data recorded.
-The supported script types are Python 2.7 and Lua 5.1 as of now.
+The supported script types are Python3, Python 2.7 and Lua 5.1 as of now.
```
Comment (honggyukim) [inline README.md:270]: For consistency, please add a space for Python version as follows.
```suggestion
The supported script types are Python 3, Python 2.7 and Lua 5.1 as of now.
```

**Review by honggyukim — COMMENTED** [review]
And please follow the @kangtegong's comment.
1. squash the commits into one
2. add `Signed-off-by` at the end of your commit message (Please read other `git log` messages)

**Comment by daeroro** [comment]
Thank you for kind comments. 
I'll amend the commit.

**Review by honggyukim — APPROVED** [review]
Thanks. LGTM.

**Review by namhyung — APPROVED** [review]
LGTM

**Pull request merged**